src/Parser/IsoParsers.php

```diff
@@ -116,17 +116,17 @@ public static function localDateRange() : PatternParser
             ->appendLiteral('/')
 
             ->startGroup()
-                ->startGroup()
-                    ->append(self::localDate())
-                ->endGroup()
+            ->startGroup()
+            ->append(self::localDate())
+            ->endGroup()
             ->appendOr()
-                ->startGroup()
-                    ->startOptional()
-                        ->appendCapturePattern(MonthOfYear::PATTERN, MonthOfYear::NAME)
-                        ->appendLiteral('-')
-                    ->endOptional()
-                    ->appendCapturePattern(DayOfMonth::PATTERN, DayOfMonth::NAME)
-                ->endGroup()
```

Comment:
Same as in tests: I do value coding standards, but I'd really miss these indents for readability. Ideas?

Reply:
Fix by https://github.com/brick/date-time/pull/53#issuecomment-1153311045